MediaBrowser.Providers/MediaInfo/AudioResolver.cs

```diff
@@ -0,0 +1,164 @@
+#pragma warning disable CA1002, CS1591
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Emby.Naming.Audio;
+using Emby.Naming.Common;
+using Jellyfin.Extensions;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.MediaEncoding;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Dlna;
+using MediaBrowser.Model.Dto;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Globalization;
+using MediaBrowser.Model.MediaInfo;
+
+namespace MediaBrowser.Providers.MediaInfo
+{
+    public class AudioResolver
+    {
+        public async Task<List<MediaStream>> GetExternalAudioStreams(
```

Comment:
It's generally frowned upon to return a list
```suggestion
        public async Task<IAsyncEnumerable<MediaStream>> GetExternalAudioStreams(
```